### DISCUSSION TITLE
Sprint-4 Discussions

### DISCUSSION THREAD

**Discussion by akhil-maddukuri** [BHAVANA1497/GatorDesk#121] [General]
@HarshithaMyadam @Nitya-vaishnavi @BHAVANA1497 I've created a new Jira board to start off with Sprint-4. We are good to start working on it.

**Comment by Nitya-vaishnavi** [comment]
@akhil-maddukuri Shall we discuss on all the backend tasks that need to be completed for this print? Since this is the final sprint, it is better that we do Integration testing and Functionality testing too.

**Reply by akhil-maddukuri** [reply]
Yes, we will do it, I just got to know that the front-end team is expecting the username and usertype upon the successful hit of the Singup and Login API.

**Reply by Nitya-vaishnavi** [reply]
We can return a json object of the user so that they can retrieve the necessary fields they want to

**Reply by akhil-maddukuri** [reply]
Sounds good. We can do that

**Comment by BHAVANA1497** [comment]
Hi @HarshithaMyadam, I see that you finished writing test cases for the signup component. Let's discuss what to include on the maintenance page and enhance this feature.

**Comment by akhil-maddukuri** [comment]
@Nitya-vaishnavi I think we are good with the backend code except for two things. 
i) Link lost and found item feature
ii)Unit tests for lost and found views

I've created user stories for these, let's focus on finishing them by EOD

**Reply by Nitya-vaishnavi** [reply]
Sounds good @akhil-maddukuri . I need a clarification regarding the link lost & found items feature. Can we connect on a quick call ?

**Reply by akhil-maddukuri** [reply]
Yes, I've sent you a meet invite on teams.
